index.html

```diff
@@ -362,7 +362,7 @@ <h5 class="year">2018</h5>
         </div>
     </section>
 
-    <section class="about-myself">
+    <section id="about-myself">
         <div class="top">
```

Comment:
- [ ] I can see you implemented some of the requested changes by the previous reviewer good job 👍 but there is something i would like to address here  kindly consider  putting the about me content into another container and giving it some width and  margin   auto and this will make the content in the center easily and  consider applying that on the other projects you will be working on it will save you time below is a hint

```
  #about-myself-container {
  
    width: 80%;
    margin: auto;

  }
```
- considering that the container you gave it has an id of `about-myself-container`